humio.go

```diff
@@ -102,17 +102,20 @@ func (c *client) do(req *http.Request) (*http.Response, error) {
 	if err != nil {
 		return nil, err
 	}
+	logs, _ := zap.NewProduction()
```

Comment:
I would recommend using go.lunarway.com/zlog. It doesn't require this setup and uses zap underneath =D

Reply:
As such you can just do zlog.Info to log

Reply:
That is a private library and this project is OSS so not available. 😩

Reply:
Ah dang it.

Reply:
In that case, I would recommend making a global zap logger, so you don't have to create a new one all the time =D

https://jimkang.medium.com/golang-global-logging-using-uber-zap-dfbbb925396a

Reply:
Or store one instance on the client for reuse 👍

Reply:
yep.

Reply:
Np, I will make a global logger 👍